stratum/hal/bin/barefoot/build-bf-sde.sh

```diff
@@ -150,7 +158,8 @@ else
     echo "SDE version: ${SDE_VERSION}"
 fi
 
-if [[ $SDE_VERSION == "9.7.0" ]]; then
+if [[ ! $(numeric_version "$SDE_VERSION") < $(numeric_version "9.7.0") ]]; then
```

Comment:
```suggestion
if [[ $(numeric_version "$SDE_VERSION") -ge $(numeric_version "9.7.0") ]]; then
```

https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html